src/main/java/com/teragrep/pth_06/planner/bloomfilter/FilterFromRecordToCategoryTableConsumer.java

```diff
@@ -43,126 +43,70 @@
  * Teragrep, the applicable Commercial License may apply to this file if you as
  * a licensee so wish it.
  */
-package com.teragrep.pth_06.planner;
+package com.teragrep.pth_06.planner.bloomfilter;
 
-import com.teragrep.blf_01.Token;
-import org.apache.spark.util.sketch.BloomFilter;
-import org.jooq.*;
+import org.jooq.DSLContext;
+import org.jooq.Field;
+import org.jooq.Record;
+import org.jooq.Table;
 import org.jooq.impl.DSL;
 import org.jooq.types.ULong;
 
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
-import java.io.UncheckedIOException;
-import java.util.regex.Pattern;
+import java.util.Objects;
+import java.util.function.Consumer;
 
 import static com.teragrep.pth_06.jooq.generated.bloomdb.Bloomdb.BLOOMDB;
 import static org.jooq.impl.SQLDataType.BIGINTUNSIGNED;
 
-/**
- * Filter types of a table that can be inserted into the tables category table
- */
-public final class TableFilters {
+public final class FilterFromRecordToCategoryTableConsumer implements Consumer<Record> {
```

Comment:
please use object way instead of functional way for producing an iterator meaning for loop instead of a consumer

Reply:
refactored to use for loop